### ISSUE TITLE
SlingPostProcessor Example

### ISSUE THREAD

**Issue by dherges** [Adobe-Consulting-Services/acs-aem-samples#36]
Hi dudes,

i a veeeery fascinating project, stranger things have happened because some people didn't know about SlingPostProcessor ... https://sling.apache.org/apidocs/sling7/org/apache/sling/servlets/post/SlingPostProcessor.html

Maybe we should get a very  veeerrrrrry nice example into ACS samples :-)

I'll try to open up a PR asap...

**Comment by davidjgonzalez** [comment]
+2! 

There is a PropertyMergePostProcessor in ACS Commons that could provide the framework for a POST processor.

In typical ACS Samples fashion, we should ad guidance in the the class comments on when to use them and when not to, and what the potential downsides are (they are called on all invocations of the Sling POST Servlet)

**Comment by dherges** [comment]
> (they are called on all invocations of the Sling POST Servlet)

yeah, we used it by writing a simple `accepts(SlingHttpServletRequest)` that checks whether the PostProcessor should/must be triggered ... usually like `if("my/resource-type".equals(request.getResource().getResourceType())` ...

**Comment by davidjgonzalez** [comment]
Perfect for the sample!!
